### PR TITLE
New PowerBIDedicated SDK

### PR DESCRIPTION
**Pull request by selasagi**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X ] Title of the pull request is clear and informative.
- [X ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Microsoft Azure PowerBI Dedicated Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for PowerBI Dedicated Resources.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```
Comment (shahabhijeet) [inline src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs]: @selasagi make it compatible with nuget version to 0.9.0 for Assembly and fileversion

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please add link for your Spec PR
Also describe what is going in this PR.
Yes I know you are generating your SDK, but this PR should reflect what is added in this PR.

File: src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (selasagi) [reply src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs]: The link to the spec - https://github.com/Azure/azure-rest-api-specs/tree/current/specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/2017-10-01/powerbidedicated.json
it is merged. Thanks

File: src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyFileVersion("1.0.0.0")]
```
Comment (shahabhijeet) [inline src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs]: Assembly file and assembly version should match with nuget version at least when you are not non-preview yet.
Once you become preview, only AssemblyFile version matches nuget version and AssemblyVersion revs up once you update major version.

File: src/SDKs/PowerBIDedicated/PowerBIDedicated.Tests/Helpers/PowerBIDedicatedTestUtilities.cs [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+namespace DedicatedServices.Tests.Helpers
+{
+    public static class PowerBIDedicatedTestUtilities
+    {
+        private static HttpClientHandler Handler = null;
+        private static Uri testUri = new Uri("https://api-dogfood.resources.windows-int.net/");
+
+        // These should be filled in only if test tenant is true
+#if DNX451
```
Comment (shahabhijeet) [inline src/SDKs/PowerBIDedicated/PowerBIDedicated.Tests/Helpers/PowerBIDedicatedTestUtilities.cs]: @selasagi we do not support DNX target anymore. Please use FullNetFx (which is targeting full desktop version) or everything else just targets .NET standard 1.4

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Pull request closed without merging**